.github/workflows/ci.yml

```diff
@@ -146,3 +146,24 @@ jobs:
     - name: Create source and wheel dist
       run: |
         python -m build
+
+  server-package:
+    # This job checks if we can build our server package
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./server
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build the Docker image
+        run: |
+          docker buildx build -t basyx-python-sdk-http-server .
```

Comment:
Why are we using `docker buildx build` instead of `docker build` here?
As far as my quick search went, buildx is used for multi-platform building, which I suppose makes sense in the context of a CI, however the way it's used here, we only build for the default Ubuntu platform (which I guess is `linux/amd64` (x86_64)?) 

Maybe it makes sense to build for the most common platforms, being (to my knowledge):
- linux/amd64 (x86 chips)
- linux/arm64 (ARM chips)

I'm no expert (first time I see `buildx`), but it could look like this: 

```
docker buildx build --platform linux/amd64,linux/arm64 -t basyx-python-server .
```

Considering that Windows Docker installations by default now work with WSL2, I think we don't need to check for windows platforms.

Reply:
Hmm, I do not really think it is necessary to support multiple system architectures with our docker server for now. But you know more about our users, do you think we should?